Description: Avoid use of git for resolving upstream version
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2016-03-06

--- a/version.sh
+++ b/version.sh
@@ -1,7 +1,4 @@
 if test "x$SASSC_VERSION" = "x"; then
-  SASSC_VERSION=`git describe --abbrev=4 --dirty --always --tags 2>/dev/null`
-fi
-if test "x$SASSC_VERSION" = "x"; then
   SASSC_VERSION=`cat VERSION 2>/dev/null`
 fi
 if test "x$SASSC_VERSION" = "x"; then
